ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContextSpec.scala

```diff
@@ -67,6 +67,31 @@ class CommitContextSpec extends WordSpec with Matchers {
     }
   }
 
+  "readAllFiltered" should {
+    "return keys matching the predicate and mark them as accessed" in {
+      val expectedKey1 = DamlStateKey.newBuilder.setContractId("a1").build
```

Comment:
Minor: an utility here could improve readability (keys are built thrice only in this test).